### PR TITLE
Homework Podstawy C++ #2

### PR DESCRIPTION
**Pull request by Ian64**
Authors: @malinaq3, @Ian64

### CODE REVIEW

**Comment by piezol** [comment]
Only commit the files required to do the task - it's impossible to do code review right now

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -3,35 +3,66 @@ name: C++ with CMake CI
 on: [push, pull_request]
```
Comment (piezol) [inline .github/workflows/main.yml:3]: The target of this pull request is wrong: we are supposed to create PR to 'module2', not the master.

Comment (Ian64) [reply .github/workflows/main.yml]: Fixed, thanks for noticing that.

File: module2/homework/generateSequence/generateSequence.hpp [hunk 2]
```diff
@@ -2,6 +2,17 @@
 #include <vector>
 
 std::vector<int> generateSequence(int count, int step) {
-    // TODO: Implement me :)
-    return {};
+    if (count <= 0) {
+        return {};
+    }
+    else {
```
Comment (nauka-programowania-MA) [inline module2/homework/generateSequence/generateSequence.hpp:8]: else not needed

**Comment by ziobron** [comment]
+4 XP addEven
+6 XP NwwNwd
+5 XP maxOfVector
+5 XP generateSequence
+8 XP delivery before 31.05
+3 XP cooperation

{
  "requirement_ids": [1119, 1631, 1119, 1633, 1119, 1634, 1119, 1635, 1636],
  "points": [2, 4, 2, 6, 2, 5, 2, 5, 3],
  "users": ["malinaq3", "Ian64"]
}

**Pull request closed without merging**